qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -473,6 +473,41 @@ def parse_offset_get(s):
 
         self.connect_message()
 
+    def increment_sensitivity(self):
+        """
+        Increment the sensitivity setting of the lock-in. This is equivalent
+        to pushing the sensitivity up button on the front panel. This has no
+        effect if the sensitivity is already at the maximum.
+
+        Returns:
+            Whether or not the sensitivity was actually changed.
+        """
+        return self._change_sensitivity(1)
+
+    def decrement_sensitivity(self):
+        """
+        Decrement the sensitivity setting of the lock-in. This is equivalent
+        to pushing the sensitivity down button on the front panel. This has no
+        effect if the sensitivity is already at the minimum.
+
+        Returns:
+            Whether or not the sensitivity was actually changed.
+        """
+        return self._change_sensitivity(-1)
+
+    def _change_sensitivity(self, dn):
+        n = int(self.ask('SENS?'))
```

Comment:
I'd like to suggest reusing the `sensitivity` parameter, so that duplicated bare SCPI commands are not scattered around the driver (unless really needed).

So, in order to do that, what about the following:
```python
self.sensitivity.get()  # or self.sensitivity()
n = self.sensitivity.raw_value
```
and for setting the new sensitivity value below, just use the `n_to`:
```python
self.sensitivity.set(n_to(n + dn))  # or self.sensitivity(n_to(n + dn))
```
What do you think?

Reply:
Done. This does make sense, I didn't know the raw value was accessible in this way.